neptune/new/types/atoms/artifact.py

```diff
@@ -0,0 +1,39 @@
+#
+# Copyright (c) 2021, Neptune Labs Sp. z o.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from dataclasses import dataclass
+from typing import TypeVar, TYPE_CHECKING
+
+from neptune.new.types.atoms.atom import Atom
+
+if TYPE_CHECKING:
+    from neptune.new.types.value_visitor import ValueVisitor
+
+Ret = TypeVar('Ret')
+
+
+@dataclass
+class Artifact(Atom):
+
+    value: str
```

Comment:
`hash` instead of `value` might be more meaningful

Reply:
In addition, I don't think hash is actually a "value" like str/boolean/etc. It's something internally required to operate on "Artifact" instead of a being. Check GitRef as an example :wink: